vspd.go

```diff
@@ -74,6 +82,31 @@ func run(ctx context.Context) error {
 	}
 	defer db.Close()
 
+	//Check if adminPass Hash is set in db.
+	hash, err := db.GetAdminHash()
+
+	//Ensure adminpass option is set
+	if cfg.AdminPass == "" && err != nil {
+		log.Error("Config error: the adminpass option is not set")
```

Comment:
Does all errors mean the adminpass is not set? Also, it's not adminpass option anymore as it isn't a config property, more like `admin password is not yet set`.